### ISSUE TITLE
Daily rolling file not using system time

### ISSUE THREAD

**Issue by michieltjampens** [tinylog-org/tinylog#217]
I might be mistaken, but i assume this uses the computer clock. Could it be possible to define an offset? Or maybe choose between UTC and localtime/systemtime?

**Comment by pmwmedia** [comment]
Supported examples:

```
daily: 00:00@UTC
daily: 00:00@GMT+01:00
daily: 00:00@GMT-01:00
daily: 00:00@America/New_York
daily: 00:00@Europe/Berlin
```

Of course, the time can be still defined without any custom time zone for using the local system time zone.

**Edit:** Time zones can be also used for the monthly policy.

**Comment by pmwmedia** [comment]
Will be released with the second milestone of tinylog 2.4

**Comment by michieltjampens** [comment]
Thanks for the quick reply/resolution!
